### PR TITLE
Debug Total Doctors count on Facility page

### LINKED ISSUE #4793: Total Dccotors count on Facility Page doesn't get updated
**Describe the bug**
When I add a new doctor type, the count of the total doctors doesn't get updated unless we reload the page.

**To Reproduce**

1. Go to https://care.coronasafe.in/facility/2bfdbbe6-0516-4269-9d17-860efd705ffb
2. Add a New Doctor Type
3. See error

**Expected behavior**
The Total Doctors count should be updated every time the doctor is added.

**Screenshots**
image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by Pranshu1902**
Fixes #4793 
The total doctor's count gets updated every time a new doctor is added.
https://care.coronasafe.in/facility/2bfdbbe6-0516-4269-9d17-860efd705ffb
image

### CODE REVIEW

File: src/Components/Facility/FacilityHome.tsx [hunk 1]
```diff
@@ -774,6 +774,14 @@ export const FacilityHome = (props: any) => {
...
               if (doctorRes && doctorRes.data) {
                 setDoctorData(doctorRes.data.results);
+                // update total doctors count
+                let totalCount = 0;
+                doctorRes.data.results.map((doctor: DoctorModal) => {
+                  if (doctor.count) {
+                    totalCount += doctor.count;
+                  }
+                });
+                setTotalDoctors(totalCount);
```
Comment (rithviknishad) [inline src/Components/Facility/FacilityHome.tsx]: @Pranshu1902 use `reduce` instead of `map`

**Review by rithviknishad — CHANGES_REQUESTED** [review]

File: src/Components/Facility/FacilityHome.tsx (hunk 1 above)
Comment (rithviknishad) [reply src/Components/Facility/FacilityHome.tsx]: https://stackoverflow.com/a/16751601/7887936

Comment (Pranshu1902) [reply src/Components/Facility/FacilityHome.tsx]: Done

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Comment by khavinshankar** [comment]
@rithviknishad can you rereview this

**Review by rithviknishad — APPROVED** [review]

**Pull request merged**